extension/src/service/web-component-service.js

```diff
@@ -24,7 +24,14 @@ function cancelOrRefund (cancelOrRefundRequestObj) {
   return callAdyen(`${config.adyen.legacyApiBaseUrl}/cancelOrRefund`, cancelOrRefundRequestObj)
 }
 
-function manualCapture (manualCaptureRequestObj) {
+function manualCapture (chargePendingTransaction, authorizationTransactionInteractionId) {
+  const manualCaptureRequestObj = {
```

Comment:
Could you please put those lines into `execute` method to be able to follow to contract like other Adyen calls?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/341/commits/87dd9e28d8b0b70a5ee1bc232f52f2b12cb4ce9f